axelor-business-project/src/main/java/com/axelor/apps/businessproject/service/InvoicingProjectService.java

```diff
@@ -487,4 +489,26 @@ public void generateAnnex(InvoicingProject invoicingProject) throws AxelorExcept
     }
     reportSettings.toAttach(invoicingProject).generate();
   }
+
+  @Transactional(rollbackOn = {AxelorException.class, Exception.class})
+  public InvoicingProject generateInvoicingProject(Project project) {
```

Comment:
override this method on business production module too, to handle case for production order